torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6068,23 +6068,23 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            op=torch.lu,
            dtypes=floating_and_complex_types(),
            supports_inplace_autograd=False,
+           # we use in-place operations which cannot be avoided.
+           # This causes vmap failures, hence we skip batched gradient checks
+           check_batched_grad=False,
```

Comment:
why doesn't batched gradient computation work for in-place operations? cc. @zou3519

Reply:
This is fine for now. See the note at https://github.com/pytorch/pytorch/blob/48af9de92ffae89618a22291738189e3110d14ed/aten/src/ATen/BatchedFallback.cpp#L123 for why some in-place operations make vmap sad

Reply:
Unfortunately, since the grad is defined block-wise, I do not know how to avoid in-placeness here.

Reply:
In the future we'll probably add operations like "narrow_copy" that combine the narrow + in-place into a single out-of-place op.